packages/actions-global/tsconfig.declarations.json

```diff
@@ -1,5 +1,6 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": { "declaration": true, "isolatedModules": false },
-  "exclude": ["**/*.spec.ts", "**/*.spec.tsx"]
+  "exclude": ["**/*.spec.ts", "**/*.spec.tsx"],
+  "include": []
```

Comment:
The "fix" that I found is basically to override the `include` list of the base `tsconfig.json`, and only include stuff that is necessary where needed.